glass/src/libnt/native/cpp/NetworkTables.cpp

```diff
@@ -766,6 +782,15 @@ void NetworkTablesFlagsSettings::DisplayMenu() {
   ImGui::MenuItem("Show Connections", "", m_pShowConnections);
   ImGui::MenuItem("Show Flags", "", m_pShowFlags);
   ImGui::MenuItem("Show Timestamp", "", m_pShowTimestamp);
+  if (ImGui::BeginMenu("Decimal Precision")) {
+    for (unsigned char i = 1; i <= 10; i++) {
+      if (ImGui::MenuItem(fmt::format("{}", i).c_str(), nullptr,
```

Comment:
Might be better done as just a static array of strings?

Reply:
Wouldn't an array of integers be better then an array of strings? I think that would be less casting.

Reply:
Forgot it all ends up in a formatting string, not being passed as an integer whoops

Reply:
No.
```
static const char* items[] = {"1", "2", "3", ... };
for (unsigned char i = 1; i <= 10; i++) {
  if (ImGui::MenuItem(items[i - 1], nullptr, i == selected)) {
    selected = i;
  }
}
```

Reply:
> No.
> 
> ```
> static const char* items[] = {"1", "2", "3", ... };
> for (unsigned char i = 1; i <= 10; i++) {
>   if (ImGui::MenuItem(items[i - 1], nullptr, i == selected)) {
>     selected = i;
>   }
> }
> ```

I think I'm just going to go with this one. Thanks for all your feedback and help on this by the way.